src/test/scala/nl/knaw/dans/easy/multideposit/CustomMatchers.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2017 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
```

Comment:
http://maven.apache.org/archives/maven-1.x/reference/maven-model/3.0.2/maven.html

inceptionYear: The year of the project's inception, specified with 4 digits. This value is used when generating copyright notices as well as being informational.